ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/validator/batch/BatchedSubmissionValidatorSpec.scala

```diff
@@ -42,7 +44,9 @@ class BatchedSubmissionValidatorSpec
       val validator = BatchedSubmissionValidator[Unit](
         BatchedSubmissionValidatorParameters.reasonableDefault,
         engine,
-        metrics)
+        metrics,
+        NoOpLedgerDataExporter,
+      )
```

Comment:
Can you refactor this into a utility method? It made sense even earlier but now with a default less (and a further parameter) is a good occasion to do so.

Reply:
Well, given this is only used for tests, I'll push it into the `apply` method and make it package-private.

Reply:
Ended up moving this method into the test class itself and making it more useful there.